tst/regression/CMakeLists.txt

```diff
@@ -49,6 +50,13 @@ list(APPEND TEST_DIRS calculate_pi)
 list(APPEND TEST_ARGS "--driver ${CMAKE_BINARY_DIR}/example/calculate_pi/pi-example \
 --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/calculate_pi/parthinput.regression")
 
```

Comment:
I would have preferred a second list to specify the number of MPI_PROCS, e.g. something like:

```
list(APPEND TEST_MPI_PROCS 1)
```

That would be passed to setup_test_mpi, in the for loop below so you are not having to call setup_test and setup_test_mpi outside of the for loop. The HDF5 case I understand though. This is more of a stylistic preference though so non blocking.

Reply:
Yeah, I think we need to address the overall logic and what we want to run where and when at some point anyway and can fix is then.
For now I just made it this here "does the job".